trade-mapping-to-ipaffs/src/test/java/uk/gov/defra/stw/mapping/toipaffs/common/ArrivalTimeMapperTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.defra.stw.mapping.toipaffs.common;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import uk.gov.defra.stw.mapping.dto.DateTime;
+import uk.gov.defra.stw.mapping.dto.DateTimeType;
+import uk.gov.defra.stw.mapping.dto.SpsCertificate;
+import uk.gov.defra.stw.mapping.dto.SpsConsignment;
+import uk.gov.defra.stw.mapping.toipaffs.common.utils.DateTimeUtil;
+import uk.gov.defra.stw.mapping.toipaffs.exceptions.NotificationMapperException;
+
+public class ArrivalTimeMapperTest {
```

Comment:
Same comments here as for ArrivalDateMapperTest

Reply:
fixed